### PR TITLE
Consolidated inventory file examples

### PR DESCRIPTION
**Pull request by ianf77**
Removed and rearranged inventory files and contents in Chapt 3 of Installation Guide

Install Guide Scenario Consolidation

https://issues.redhat.com/browse/AAP-18700

### CODE REVIEW

File: downstream/modules/platform/proc-editing-inventory-file.adoc [hunk 1]
```diff
@@ -29,7 +29,7 @@ $ cd ansible-automation-platform-setup-<latest-version>
 -----
 +
 . Open the `inventory` file with a text editor.
-. Edit `inventory` file parameters to specify your installation scenario. Use one of the supported xref:con-install-scenario-examples[Installation scenario examples] to update your `inventory` file.
+. Edit `inventory` file parameters to specify your installation scenario. Yo can use one of the supported xref:con-install-scenario-examples[Installation scenario examples] as the basic for your `inventory` file.
```
Comment (ariordan-redhat) [inline downstream/modules/platform/proc-editing-inventory-file.adoc:32]:
```suggestion
. Edit `inventory` file parameters to specify your installation scenario. You can use one of the supported xref:con-install-scenario-examples[Installation scenario examples] as the basic for your `inventory` file.
```

File: downstream/modules/platform/con-install-scenario-recommendations.adoc
Comment (msilmser) [inline downstream/modules/platform/con-install-scenario-recommendations.adoc:1]: I think we're getting a bit of back-and-forth internally between eng/pm around our DB stance. It's probably best to leave this alone until we can get an official confirmation on the future of what this looks like. I thought we did, but that conversation was reopened.

File: downstream/modules/platform/ref-single-controller-hub-eda-with-managed-db.adoc [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+-----
+[automationcontroller]
+controller.example.com
+
+[automationhub]
+automationhub.example.com
+
+[automationedacontroller]
+automationedacontroller.example.com ansible_connection=local
```
Comment (msilmser) [inline downstream/modules/platform/ref-single-controller-hub-eda-with-managed-db.adoc:26]: `ansible_connection=local` should be removed here

File: downstream/modules/platform/con-install-scenario-recommendations.adoc
Comment (ianf77) [reply downstream/modules/platform/con-install-scenario-recommendations.adoc:1]: Will most likely need another Jira for late changes.

File: downstream/modules/platform/ref-standalone-controller-hub-ext-database-inventory.adoc [hunk 3]
```diff
@@ -15,6 +16,14 @@ automationhub.example.com
 data.example.com
```
[the remark this answers is not in the document]
Comment (ianf77) [reply downstream/modules/platform/ref-standalone-controller-hub-ext-database-inventory.adoc:16]: It has one

[the remark this answers is not in the document]
Comment (msilmser) [reply downstream/modules/platform/ref-standalone-controller-hub-ext-database-inventory.adoc:16]: @ianf77 obviously my fault for not clicking the expand option 😵‍💫

**Review by msilmser — APPROVED** [review]
lgtm!

File: downstream/modules/platform/ref-single-controller-hub-eda-with-managed-db.adoc [hunk 4]
```diff
@@ -0,0 +1,108 @@
+[id="ref-single-controller-hub-eda-with-managed-db"]
+
+=  Single controller, single automation hub, and single event-driven ansible controller node with external (installer managed ) database
```
Comment (dcdacosta) [inline downstream/modules/platform/ref-single-controller-hub-eda-with-managed-db.adoc:3]: Suggest that you use the attributes for controller, hub and eda here.

File: downstream/modules/platform/ref-standalone-controller-hub-ext-database-inventory.adoc [hunk 5]
```diff
@@ -1,12 +1,13 @@
-[id="ref-standlone-hub-ext-database-inventory_{context}"]
+[id="ref-single-controller-hub-ext-database-inventory"]
 
-= Single {HubName} with external (installer managed) database
+= Single {ControllerName and single {HubName} with external (installer managed) database
```
Comment (dcdacosta) [inline downstream/modules/platform/ref-standalone-controller-hub-ext-database-inventory.adoc:3]: Missing close } bracket on ControllerName.

File: downstream/modules/platform/ref-single-controller-hub-eda-with-managed-db.adoc [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+[role="_abstract"]
+Use this example to populate the inventory file to deploy single instances of {ControllerName} and {HubName} with an external (installer managed) database.
+
+[IMPORTANT]
+====
+This scenario requires a minimum of {ControllerName} 2.4 for successful deployment of {EDAcontroller}.
+====
```
Comment (dcdacosta) [inline downstream/modules/platform/ref-single-controller-hub-eda-with-managed-db.adoc:11]: Is there any way to rewrite this so there aren't two Important messages in a row? I wonder if the first notice couldn't be added to the Intro?

**Review by dcdacosta — APPROVED** [review]
@ianf77 Great job on this!! I made a few comments, otherwise, LGTM!

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Pull request merged**